kolibri/core/discovery/utils/network/client.py

```diff
@@ -77,6 +76,8 @@ def build_for_address(cls, address, timeout=None):
 
     @classmethod
     def build_from_network_location(cls, network_location, timeout=None):
+        from kolibri.core.discovery.models import ConnectionStatus
```

Comment:
What prompted this change?

Reply:
I ran into circular dependency issues so was just a way to circumvent the issue. But I think this should be resolved with the changes you requested above, so i will revert.